src/main/java/duke/parser/CommandParser.java

```diff
@@ -0,0 +1,49 @@
+package duke.parser;
+
+import duke.command.AddDeadlineCommand;
+import duke.command.AddEventCommand;
+import duke.command.AddToDoCommand;
+import duke.command.Command;
+import duke.command.CompleteTaskCommand;
+import duke.command.DeleteTaskCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.HelpCommand;
+import duke.command.ListCommand;
+
+/**
+ * Parses user input.
+ */
+public class CommandParser {
+    /**
+     * Parses the command as well as any arguments entered by the user.
+     * @param fullCommand the full command entered by the user
```

Comment:
I think there should be an empty line between the description and parameter section? Noticed this is a recurring issue

Reply:
You are right. Thanks for pointing this out to me! Have fixed this. 😄